features/bootstrap/DatasetViewContext.php

```diff
@@ -431,12 +431,13 @@ public function thereShouldBeAMetaTagWithValue($arg1, $arg2, $arg3)
     public function thereShouldBeAMetaTagWithMultiplines($arg1, $arg2, \Behat\Gherkin\Node\PyStringNode $arg3 )
     {
         $expectContent = $arg3->getRaw();
+        $expectContent = implode(" ", preg_split('/\r\n|\r|\n/', $expectContent));
 
         $metaNode = $this->minkContext->getSession()->getPage()->find('xpath', "//meta[@$arg1='$arg2']");
         $actualContent = $metaNode->getAttribute('content');
-//        file_put_contents('test_expect.txt', print_r($expectContent, true));
+        $actualContent = preg_replace('/\s\s/', ' ', implode(" ", preg_split('/\r\n|\r|\n/', $actualContent)));
 
-        PHPUnit_Framework_Assert::assertEquals(implode(" ", preg_split('/\r\n|\r|\n/', $expectContent)), implode(" ", preg_split('/\r\n|\r|\n/', $actualContent)));
+        PHPUnit_Framework_Assert::assertEquals($expectContent, $actualContent, 'The content is different!');
 
     }
```

Comment:
Hi @rija ,

I used `PyStringNode` to allow input of multiple lines, then clean the `expectContent` and `actualContent` by removing the `carriage return` and `extra space`, finally `assertEquals` both of them. 
The test is passing.
What do you think about this implementation? I feel that there may be too many manipulation/operation 😅